### PR TITLE
send-google-chat-message 0.1.0

### PR DESCRIPTION
**Pull request by kyuuuyki**
### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by adborbas** [comment]
Hi @kyuuuyki , Could you please sign the CLA?

**Comment by kyuuuyki** [comment]
Hi @adborbas , sorry for that. Now signed.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- notification
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:23]: Step does not use git nor wget but pip3.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+title: Send Google Chat message
+summary: |
+  Send a Google Chat message to a private or group channel via Bot!
+description: |
+  Send a Google Chat message to a private or group channel via Bot!
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:5]: description is equal to summary

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- GOOGLE_CHAT_WEBHOOK_URL: Required
+  opts:
+    description: |
+      See https://developers.google.com/hangouts/chat/how-tos/webhooks
+    is_expand: true
+    is_required: true
+    summary: Google Chat incoming webhooks link.
+    title: Google Chat Webhook URL
+    value_options: []
+- JIRA_ORGANIZATION_NAME: Optional
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:42]: What is the relationship between Jira and Google hangouts?
It should be explained in description.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
+title: Send Google Chat message
+summary: |
+  Send a Google Chat message to a private or group channel via Bot!
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:3]: Exclamation is not needed here.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+inputs:
+- GOOGLE_CHAT_WEBHOOK_URL: Required
+  opts:
+    description: |
+      See https://developers.google.com/hangouts/chat/how-tos/webhooks
+    is_expand: true
+    is_required: true
+    summary: Google Chat incoming webhooks link.
+    title: Google Chat Webhook URL
+    value_options: []
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:41]: does not make sense for this kind of input.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- JIRA_ORGANIZATION_NAME: Optional
+  opts:
+    description: |
+      Can be extracted from JIRA link, ex. [your_organization_name].atlassian.net
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:45]: Markdown should be used here.

File: steps/send-google-chat-message/0.1.0/step.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- GOOGLE_CHAT_WEBHOOK_URL: Required
```
Comment (koral--) [inline steps/send-google-chat-message/0.1.0/step.yml:33]: Often such inputs are marked as sensitive.

**Comment by kyuuuyki** [comment]
Closed due to multiple purposes of the step. Not related to step's title and category.

**Pull request closed without merging**